src/pages/HomePage/HomePage.tsx

```diff
@@ -0,0 +1,21 @@
+// HomePage.tsx
+import { Hero, Navbar ,Restaurants, Dishes, IconsMeaning , WeekChef , AboutUs, Footer} from "../../components";
+import "./HomePage.scss";
+import {IconsData,ChefData} from "../../data";
+
+const HomePage = () => {
+  return (
+    <>
+      <Navbar />
+      <Hero/>
+      <Restaurants/>
+      <Dishes/>
+      <IconsMeaning icons={IconsData}/>
+      <WeekChef chefDetails={ChefData}/>
+      <AboutUs/>
+      <Footer/>
```

Comment:
If the footer will be on every page, you can simply place it once in the App.tsx file.